### PR TITLE
Added new command, getting issues for only logged user.

### PR DESCRIPTION
**Pull request by nemanjaSokic**
This app is something I already have been started to implement by mysefl, and I have found arcli as something I wanted to done in my app. So, I am glad to find arcli. Thank you.
I would like to continue develop arcli if it is ok to you.

### CODE REVIEW

File: client/issue.go [hunk 1]
```diff
@@ -42,6 +46,12 @@ func (c *Client) GetMyIssues() ([]Issue, error) {
 	return c.GetIssues("assigned_to_id=me")
 }
 
+// GetMyIssues fetches issues assigned only to currently logged user.
+func (c *Client) GetMyIssuesOnly() ([]Issue, error) {
+	param := fmt.Sprintf("assigned_to_id=%v", viper.GetString(config.UserID))
```
Comment (mightymatth) [inline client/issue.go:51]: Isn't this exactly the same as `c.GetIssues("assigned_to_id=me")`?

Comment (nemanjaSokic) [reply client/issue.go:51]: Flag "me" shows you every task that is assigned to you,as well as evey group you are part of.
So,if user participate in group of Development and Maintenance, flag "me" will return every task  that is assigned to these groups as well.
If you pass an id of user,you will get tasks only assigned to you,and task assigned to group(s) will be excluded.

Comment (nemanjaSokic) [reply client/issue.go:51]: Maybe next link could provide more explanation.
https://www.redmine.org/issues/13758

**Comment by mightymatth** [comment]
Closing because of new PR related to this issue.

**Pull request closed without merging**